### ISSUE TITLE
ESRI Street Map Does Not Focus When Zoomed in

### ISSUE THREAD

**Issue by cstevenson1** [leeensminger/DelDOT-NPDES-Viewer#183]
Not sure if anything can be done about it but the ESRI Street Map does not adjust well when zoomed in far. Close Bug if there is no fix.

![image](https://user-images.githubusercontent.com/16919898/31546754-aa906286-aff2-11e7-9be9-c33bb3de1edd.png)

**Comment by leeensminger** [comment]
This is how the viewer looks at 1:187 scale:

![image](https://user-images.githubusercontent.com/4103847/31892776-d8870608-b7d7-11e7-9797-6836af30c398.png)

This is how the same ESRI street map looks in ArcMap (which will be the best it is going to be) at the same scale:

![image](https://user-images.githubusercontent.com/4103847/31892822-f4782aea-b7d7-11e7-8278-145ec78e13e6.png)

We should examine to see if there's something in the viewer that is limiting the last few tiles from loading from the service.

**Comment by leeensminger** [comment]
@cstevenson1 can you confirm this was resolved?  I think it was.

**Comment by cstevenson1** [comment]
@leeensminger I don't think it was. I think it still looks a decent amount more blurry than ArcMap. 

![image](https://user-images.githubusercontent.com/16919898/58636127-9a42a700-82bd-11e9-9755-be654e6be7f4.png)
